express/styles/styles.css

```diff
@@ -695,6 +695,14 @@ main .section-wrapper .hero {
   color: initial;
 }
 
+body[data-device="mobile"] main .section-wrapper > div[data-audience="desktop"] {
+  display: none;
+}
+
+body[data-device="desktop"] main .section-wrapper > div[data-audience="mobile"] {
+  display: none;
+}
```

Comment:
I do not understand the need for those 2 rules because the code does not do something `audience` specific. And the rules are identical.

Reply:
They are not identical btw 😉